### DISCUSSION TITLE
Why the --graph flag doesn't behave like before?

### DISCUSSION THREAD

**Discussion by goustaff** [vercel/turborepo#6546] [Help] [answered]
Hello,
I'm using Turborepo since few days.

I've watched this presentation (from 2022): https://youtu.be/YX5yoApjI3M?t=885 (14:45)

I'm curious to know why the `--graph` flag doesn't generate a PNG like before?

Now, it is _just_ outputing something like this in the terminal:
```
digraph {
        compound = "true"
        newrank = "true"
        subgraph "root" {
                "[root] @repo/eslint-config#build" -> "[root] ___ROOT___"
                "[root] @repo/static-analyzer#build" -> "[root] @repo/eslint-config#build"
                "[root] @repo/static-analyzer#build" -> "[root] @repo/typescript-config#build"
                "[root] @repo/typescript-config#build" -> "[root] ___ROOT___"
                "[root] web#build" -> "[root] @repo/eslint-config#build"
                "[root] web#build" -> "[root] @repo/static-analyzer#build"
                "[root] web#build" -> "[root] @repo/typescript-config#build"
        }
}
```

Then, I can convert it to a PNG with _GraphViz_ (for example).
> `$ dot -Tpng graph.dot -o graph.png`

Now, I would like to know: why this isn't generating a PNG like before?

Thank you in advance for your answers!

**Comment by mehulkar** [comment, marked as answer]
You can get this behavior by giving the flag a value like `--graph=graph.png`https://turbo.build/repo/docs/reference/command-line-reference/run#--graph
